src/components/PostList/Post.js

```diff
@@ -10,21 +12,78 @@ import {
     Center,
     Stack,
     HStack,
-    Spacer,
+    Spacer, useDisclosure,
 } from "@chakra-ui/react";
 import {BiTrash} from "react-icons/bi";
 import {useNavigate} from "react-router";
+import CommentModal from "../CommentModal";
 
 const Post = ({post}) => {
     const navigate = useNavigate();
     const currentUser = useSelector((state) => state.currentUser);
     const dispatch = useDispatch();
+    const { isOpen, onOpen, onClose } = useDisclosure();
     const postedBy = useSelector((state) =>
         state.allUsers.find((user) => user?._id === post.userId)
     );
+    const [liked, setLiked] = useState(post.liked.includes(currentUser?._id));
+    const [disliked, setDisliked] = useState(post.disliked.includes(currentUser?._id));
+
+
+    const onClickLikePost = (e) => {
+        e.preventDefault();
```

Comment:
Is the issue that this function isn't working or that the `liked` variable isn't updating?

Reply:
it doesn't get to the backend AND there's the includes problem. As soon as i press the like it fails